packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -0,0 +1,178 @@
+import Anthropic from '@anthropic-ai/sdk';
+import { getEnvVar } from './util.js';
+import * as AI from '../index.js';
+import { Node } from '../index.js';
+import { ChatOrCompletionModelOrBoth } from './model.js';
+import {
+  AssistantMessage,
+  ChatProvider,
+  FunctionCall,
+  FunctionResponse,
+  ModelProps,
+  SystemMessage,
+  UserMessage,
+  ModelPropsWithChildren,
+} from '../core/completion.js';
+import { AIJSXError, ErrorCode } from '../core/errors.js';
+
+export const anthropicClientContext = AI.createContext<Anthropic>(
+  new Anthropic({
+    apiKey: getEnvVar('ANTHROPIC_API_KEY', false),
+  })
+);
+
+type ValidCompletionModel = never;
+/**
+ * The set of valid Claude models.
+ *
+ * @see https://docs.anthropic.com/claude/reference/complete_post.
+ */
+type ValidChatModel =
```

Comment:
I'm coming to the conclusion that this is not a good pattern. I don't think we want to have to update our SDK every time some vendor releases a new model.

Reply:
I agree that's a pain, but the type completion is realllly nice. :smile:

The types are just a static check that can be bypassed, so if someone wanted to use a model that we didn't have in this list, they could just do:

```tsx
// @ts-expect-error
model='gpt-4-5-turbo'
```

which is gross, but at least it's not a hard blocker.